### ISSUE TITLE
No recovery email/sms is being sent

### ISSUE THREAD

**Issue by nleinich** [keybase/keybase-issues#4154]
Since I forgot my password for my account (username: nicoleleinich), I'd like to reset my account but even though I clicked multiple times on the "Start account reset" or "Send again" button, I still didn't get any emails nor any sms with instructions. 
I retried this both on the mac application and on the website.
Could you please help me with that?

**Comment by felinae98** [comment]
+1，my username: felinae98

**Comment by ragecc** [comment]
Same here username ragecc My phone was the only way I could get in. I had to reset it and didn't have a chance to backup anything. No email or anything is being sent. I have checked spam and more than everything. I can prove its me.

**Comment by mentalisttraceur** [comment]
Same issue + duplicate of #4160 ... starting to seem likely that this is a long-neglected bug....

**Comment by ragecc** [comment]
Yeah. I never even got a middle finger or any kind of reply. Its been since about june 2021 since I made the account. I wonder how many other people wont ever use it because of being locked out. Im convinced there is nothing but the same phone connected to the username that I still use but had to factory reset so it thinks its different. Why offer recovery if the email or phone number is never connected to the account to recover???
